### DISCUSSION TITLE
Permission Manager Plugin is looking for table users instead of admins

### DISCUSSION THREAD

**Discussion by mobi35** [Laravel-Backpack/community-forum#698] [Q&A (Help)]
in permissionsmanager.php, I already set the model to Admin::class, but it's still looking for User::class. What could be the reason? I'm using backpack 5.5. I can see the data from Admin::class in the list, but I cannot edit or create a new one.

**Comment by jcastroa87** [comment]
Hello @mobi35 i will close this because is duplicate with https://github.com/Laravel-Backpack/PermissionManager/issues/340
